sdk/trace/span.go

```diff
@@ -298,6 +304,36 @@ func typeStr(i interface{}) string {
 	return fmt.Sprintf("%s.%s", t.PkgPath(), t.Name())
 }
 
+func recordStackTrace(err error) string {
+	var stackTrace []uintptr
+	var convertedStackTrace string
+
+	if err, ok := err.(interface {
+		StackTrace() errors.StackTrace
+	}); ok {
+		for _, frame := range err.StackTrace() {
+			stackTrace = append(stackTrace, uintptr(frame))
+		}
+	}
```

Comment:
I would rather not include support for a package that is going [into maintenance mode and plans to be replaced by the Go2 errors implementation](https://github.com/pkg/errors#roadmap). Is there a reason we could not just add stacktraces with the `runtime` package alone?

Reply:
No I think `runtime` package alone should be able to capture stack traces. I think it's fair to not include this since especially it is going into maintenance mode.